src/karmen_frontend/src/components/utils/collapsible.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from "react";
+
+const Collapsible = ({
+  children,
+  collapsedStateText,
+  expandedStateText,
+  isInForm,
+}) => {
+  const [expanded, setExpanded] = useState();
+
+  const toggleTo = (evt, val) => {
```

Comment:
`toggleTo` should be `toggle` instead. It's binary, the target value is always the opposite of the current one (=`!expanded`). This will simplify the `onClick` handlers as well.

Reply:
Done.